encoder/vaapiencoder_hevc.cpp

```diff
@@ -1290,7 +1290,7 @@ void VaapiEncoderHEVC::shortRfsUpdate(const PicturePtr& picture)
     m_shortRFS.inter_ref_pic_set_prediction_flag = 0;
 
     if (intraPeriod() > 1 && m_refList0.size()) {
-        m_shortRFS.num_negative_pics         = 1;
+        m_shortRFS.num_negative_pics         = m_refList0.size();
```

Comment:
this wired, why we need set the reference frame both a negative and positive.  could we just set it as positive?